codec/src/main/java/io/netty/handler/codec/base64/Base64.java

```diff
@@ -312,6 +316,11 @@ public static ByteBuf decode(
         return new Decoder().decode(src, off, len, allocator, dialect);
     }
 
+    // package-private for testing
+    static int len34(int len) {
+        return (len * 3) >>> 2;
```

Comment:
This don't fix overflow. `(2147483647 * 3) >>> 2 == 536870911`, but must be `1610612735`.
I already offered an [option earlier](https://github.com/netty/netty/pull/6404#discussion_r101889166):

`int len34 = len - (len >>> 2);`